doc/changelog.rst

```diff
@@ -2,4 +2,4 @@
 CHANGELOGS
 ============
 
-.. include:: ../CHANGELOG.rst
```

Comment:
I cannot see what is being changed here.

Reply:
It was missing one empty line at the end, now it was added